kotest-assertions/kotest-assertions-json/src/jvmMain/kotlin/io/kotest/assertions/json/paths/jsonMatchers.kt

```diff
@@ -0,0 +1,52 @@
+package io.kotest.assertions.json.paths
+
+import io.kotest.assertions.json.beJsonArray
+import io.kotest.assertions.json.beJsonObject
+import io.kotest.assertions.json.beValidJson
+import io.kotest.assertions.json.matchJson
+import io.kotest.matchers.and
+import io.kotest.matchers.paths.aFile
+import io.kotest.matchers.paths.exist
+import io.kotest.matchers.should
+import java.nio.file.Path
+import kotlin.io.path.readText
+
+fun Path.shouldBeEmptyJsonArray(): Path {
```

Comment:
Is there any chance we prefer to have these as a simple alias to files? `this.toFile().shouldBeEmptyJsonArray()'.

We save ourselves one implementation this way

Reply:
I'd rather do `File` -> `Path` instead of `Path` -> `File`, tbh

for two reasons:
- not all `Path`s can be represented by `File`s (ones from the default provider (`file://`) can be represented as `File`s, however you can create paths from different providers including, but not limited to, `jar://`, `jimfs://`, and more)
- `Path` is the "new IO" and intended to mostly replace `File`